Packs/Core/ReleaseNotes/3_4_55.md

```diff
@@ -0,0 +1,8 @@
+<~PLATFORM>
```

Comment:
The command is hidden you should not include this description in the release notes and remove the README as well this comment should be inside a comment clause and the release notes should be Metadata improvements..